packages/application-shell/src/components/theme-switcher/new-design-release-info-dialog.tsx

```diff
@@ -0,0 +1,36 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import { InfoDialog } from '@commercetools-frontend/application-components';
+import mcRedesignAnnouncement from '@commercetools-frontend/assets/images/mc-redesign-announcement.svg';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import messages from './messages';
+
+type TNewDesignReleaseInfoDialog = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+const NewDesignReleaseInfoDialog = (props: TNewDesignReleaseInfoDialog) => {
+  const intl = useIntl();
+  return (
+    <InfoDialog
+      title={intl.formatMessage(messages.title)}
+      isOpen={props.isOpen}
+      onClose={props.onClose}
+      size="l"
+    >
+      <Spacings.Stack>
```

Comment:
Since @FilPob mentioned there should be `32px` of separation between the image and the text, I would expect us to use `scale="xl"` here.

```suggestion
      <Spacings.Stack scale="xl">
```